rsconnect_jupyter/static/connect.js

```diff
@@ -470,6 +470,16 @@ define([
         '            <span class="help-block"></span>',
         "        </div>",
         "    </div>",
+        '    <div class="form-group">',
+        '        <label>',
+        '          <input id="include_files" name="include_files" type="checkbox">',
```

Comment:
Should probably use hyphens rather than underscores in element param values here.